integration/kubernetes/helm/alluxio/templates/alluxio-format-master.yaml

```diff
@@ -0,0 +1,79 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{- $masterCount := int .Values.resources.masterCount }}
```

Comment:
@madanadit I have to include some checks to make sure the master can work with the correct config, especially if journaling in HDFS. However the checks become duplicate to what we have in `alluxio-master.yaml` template.

Reply:
can we use the helpers file https://github.com/Alluxio/alluxio/blob/master/integration/kubernetes/helm/alluxio/templates/_helpers.tpl?

Reply:
I think the helper file can be applied to generate templates corresponding to sections like Volumes, but the change will be rather big. Can we refactor the duplicate lines in a separate PR?